tests/letstest/scripts/test_new_leauto.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -xe
+
+# $OS_TYPE $PUBLIC_IP $PRIVATE_IP $PUBLIC_HOSTNAME $BOULDER_URL
+# are dynamically set at execution
+
+cd letsencrypt/letsencrypt
+VERSION=`sed -n "s/^__version__\s*=\s*[\"']\(.*\)[\"']/\1/p" __init__.py`
+cd ../letsencrypt_auto
+
+RunLetsencryptAuto() {
+    OUTPUT=`./letsencrypt-auto -v --debug --version 2>&1`
+
+    if [[ $OUTPUT != *"$VERSION"* ]] ; then
```

Comment:
I suspect I'm  misreading something, but this *seems* to say, did letsencrypt-auto produce the same version string that we found in `letsencrypt/__init__.py`.  But wouldn't sed find the current dev version and the `letsencrypt-auto` the latest public release?  Or do we run this on release tags/branches?